File: SoloLearn Codes/Jungle Camping/JungleCamping.cs
using System;
using System.Collections.Generic;

namespace SoloLearn
{
    class Program
    {
        static void Main(string[] args)
        {
            var d = new Dictionary<string, string>() {
                {"Grr", "Lion"}, {"Rawr", "Tiger"},  {"Ssss", "Snake"}, {"Chirp", "Bird"}
            };
            string[] ns = Console.ReadLine().Split();
            foreach (string n in ns) Console.Write(d[n] + ' ');
        }
    }
}

File: SoloLearn Codes/Jungle Camping/jungle_camping.swift
var ns = readLine()!.split(separator: " ")
for n in ns {
    switch n {
        case "Grr":
            print("Lion", terminator: " ")
        case "Rawr":
            print("Tiger", terminator: " ")
        case "Ssss":
            print("Snake", terminator: " ")
        case "Chirp":
            print("Bird", terminator: " ")
        default:
            break
    }
}